### PR TITLE
Fixes demon icon, typo, limits wizard to summoning of demon

### LINKED ISSUE #10045: Slaughter demon typo
"You being to feast on Tony Rosso. You can not move while you are doing this."

>You being

@KorPhaeron

### LINKED ISSUE #10058: Healing/reviving a slaughter demon via player panel has weird effects.
The slaughter demon will be turned into ye-olde orange human sprite (orange skin, green eyes) but cannot turn. It retains the same path, health, sound effect, attacks, and blood movement abilities.

@KorPhaeron

### PR DESCRIPTION
**Pull request by ghost**
Fixes #10045
Fixes #10058

Wizard can now only summon a single demon.

### CODE REVIEW

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -302,6 +302,15 @@
...
 	log_name = "BB"
+	var/limit = 0
+
+/datum/spellbook_entry/item/bloodbottle/Buy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
+	if(..())
+		limit = 1
+	return 1
+
+/datum/spellbook_entry/item/bloodbottle/CanBuy(var/mob/living/carbon/human/user,var/obj/item/weapon/spellbook/book)
+	return ..() && !limit
```
Comment (Anonus) [inline code/game/gamemodes/wizard/spellbook.dm]: Make this work for everything, not just bottles.

**Comment by KorPhaeron** [comment]
I didnt see a reason to limit the other items. If the wizard wants to spawn 5 scrying orbs then whatever

**Comment by RemieRichards** [comment]
@KorPhaeron the way you've done it now is Snowflake, is what @Anonus is getting at, it would be for the best that all spellbook_entries be able to supply and check against a limit.

**Comment by daman997** [comment]
literally the worst change ever worse than catpeople actually not but still so bad we need more 5 demon wizards

**Comment by phil235** [comment]
I've made another PR that fixes the problems mentionned above. Closing this one

**Pull request closed without merging**